include/cudaq/Optimizer/CodeGen/QIRFunctionNames.h

```diff
@@ -29,6 +29,8 @@ constexpr static const char NVQIRInvokeWithControlBits[] =
     "invokeWithControlQubits";
 constexpr static const char NVQIRInvokeRotationWithControlBits[] =
     "invokeRotationWithControlQubits";
+constexpr static const char NVQIRInvokeWithControlRegisterOrBits[] =
+    "invokeWithControlRegisterOrQubits";
```

Comment:
Will this new function ever end up in the Base Profile or Adaptive Profile QIR? I'm thinking it won't because I believe multi-control decomposition would've eliminated all of these, but I figured I'd double check.

Reply:
You're right, it should not due to the multi-control decomp pass.